.github/actions/aws-ci/action.yml

```diff
@@ -35,9 +41,10 @@ runs:
         AWS_ACCESS_KEY_ID: ${{ inputs.aws-access-key-id }}
         AWS_SECRET_ACCESS_KEY: ${{ inputs.aws-secret-access-key }}
         AWS_ACCOUNT_TYPE: ${{ inputs.aws-account-type }}
+        CLOUDBEAT_CONFIG: ${{ inputs.cloudbeat-config }}
       shell: bash
       run: |
-        ./cloudbeat -c deploy/aws/cloudbeat-aws.yml -d '*' &
+        ./cloudbeat -c ${{ env.CLOUDBEAT_CONFIG }} -d '*' &
```

Comment:
```suggestion
        ./cloudbeat -c "$CLOUDBEAT_CONFIG" -d '*' &
```

Reply:
Resolved by having a separate action for asset inventory.